doc/ref_model/chapters/chapter05.rst

```diff
@@ -169,6 +169,41 @@ Tracing                                Jaeger
 
 **Table 5-7a:** Service examples.
 
+Platform Services - Load Balancer Requirements
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The table below specifies a set of requirements for the Load Balancer platform service.
+
+========== ===================================================================== ============================================================================
+Reference  Requirement                                                           Notes
+========== ===================================================================== ============================================================================
+pas.lb.001 The Load Balancer must support resource scaling	
+pas.lb.002 The Load Balancer must support resource resiliency	
+pas.lb.003 The Load Balancer must support scaling and resiliency in the local    Define local environment: within a subnet, tenant network, Availability 
+           environment                                                           Zone of a cloud, ...
+pas.lb.004 The Load Balancer should support resiliency in the global environment	
+pas.lb.005 The Load Balancer must support OSI Level 3/4 load-balancing and may   OSI Level 3 load-balancing decision on the source and destination IP 
+           support Level 7 Load Balancing                                        addresses and OSI Level 4 TCP port numbers. <br>OSI Level 7 (application
```

Comment:
Please do not use raw html, like <br>. You can enforce new lines with the [block syntax](https://docutils.sourceforge.io/docs/user/rst/quickref.html#line-blocks) as it is explained [here](https://stackoverflow.com/questions/13504595/multi-line-table-cell-in-restructuredtext).

Reply:
@CsatariGergely Thanks - Done. Please review.